js/ui/appDisplay.js

```diff
@@ -1191,7 +1191,7 @@ const FolderView = new Lang.Class({
 
     nRowsDisplayedAtOnce: function() {
         let [availWidthPerPage, availHeightPerPage] = this._getPageAvailableSize();
-        let maxRows = this._grid.rowsForHeight(availHeightPerPage) - 1;
```

Comment:
I'm a bit surprised to see that this hasn't been a problem for us now and that simply removing it doesn't cause any regression, but I've tested this branch and it seems to work so I guess it might be fine considering the other downstream changes we made so far.

In any case it would be interesting to try to understand why that `- 1` was there in the first place.